.eslintrc.js

```diff
@@ -88,6 +88,7 @@ module.exports = {
       },
     ],
     'va/correct-apostrophe': 1,
+    'va/prefer-web-component-library': 0,
```

Comment:
This rule is off globally because if left on, husky will run the autofix on any changed file before commit and an error will be thrown since `no-unused-vars` will find unused imports:

![image](https://user-images.githubusercontent.com/2008881/150893472-d2c8691f-d254-4ba8-9989-596dfb17cec8.png)

We want to be careful not to be annoying if someone is trying to make a completely unrelated change. There's some background on this (and a slack thread) in https://github.com/department-of-veterans-affairs/vets-website/pull/17442.